src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/db/Recall.kt

```diff
@@ -111,8 +112,8 @@ data class Recall(
   @Enumerated(STRING)
   val previousConvictionMainNameCategory: NameFormatCategory? = null,
   val hasDossierBeenChecked: Boolean? = null,
-  val previousConvictionMainName: String? = null,
   // MD: ideally this would be UserId, but hibernate/postgres does not make this easy :-(
```

Comment:
Comment does not make sense after edit?

Reply:
That's interesting. wonder how it got reordered!